packages/react/src/components/dropdown/combobox/Combobox.tsx

```diff
@@ -376,6 +380,7 @@ export const Combobox = <OptionType,>(props: ComboboxProps<OptionType>) => {
         !showToggleButton && styles.noToggle,
         hasFocus && selectedItems.length > 0 && styles.adjustSpacing,
         props.icon && props.multiselect && styles.inputWithIcon,
+        clearable && !props.multiselect && selectedItem && styles.withClearButton,
```

Comment:
The three first statements are the same as showClearButtonForSingleSelect

Reply:
- [ ] Use showClearButtonForSingleSelect